prov/usnic/src/fi_ext_usnic.h

```diff
@@ -78,4 +78,15 @@ struct fi_usnic_ops_av {
 	int (*get_distance)(struct fid_av *av, void *addr, int *metric);
 };
 
+/*
+ * usNIC-specific provider completion status errnos
+ */
+enum {
+	FI_EXT_USNIC_COMPSTAT_SUCCESS,
+	FI_EXT_USNIC_COMPSTAT_ERROR_CRC,
```

Comment:
Works for me.  I wasn't sure how closely you wanted to stick to the list of errors in Linux's/glibc's errno.h.  `TRUNC` seems another likely candidate for a new error code.  None of the existing ones seemed appropriate.